ibm_catalog.json

```diff
@@ -83,96 +84,170 @@
-              {
-                "key": "ibmcloud_api_key",
-                "required": true,
-                "type": "password"
-              },
-              {
-                "key": "provider_visibility",
-                "options": [
-                  {
-                    "displayname": "private",
-                    "value": "private"
-                  },
-                  {
-                    "displayname": "public",
-                    "value": "public"
-                  },
-                  {
-                    "displayname": "public-and-private",
-                    "value": "public-and-private"
-                  }
-                ]
-              },
-              {
-                "key": "use_existing_resource_group"
-              },
-              {
-                "key": "resource_group_name"
-              },
-              {
-                "key": "prefix",
-                "required": true,
-                "description": "Prefix to add to all resources created by this solution. To not use any prefix value, you can enter the string `__NULL__`."
-              },
-              {
-                "key": "watsonx_data_name"
-              },
-              {
-                "key": "region",
-                "required": true,
-                "default_value": "us-south",
-                "options": [
-                  {
-                    "displayname": "Dallas (us-south)",
-                    "value": "us-south"
-                  },
-                  {
-                    "displayname": "Frankfurt (eu-de)",
-                    "value": "eu-de"
-                  },
-                  {
-                    "displayname": "London (eu-gb)",
-                    "value": "eu-gb"
-                  },
-                  {
-                    "displayname": "Sydney (au-syd)",
-                    "value": "au-syd"
-                  },
-                  {
-                    "displayname": "Tokyo (jp-tok)",
-                    "value": "jp-tok"
-                  }
-                ]
-              },
-              {
-                "key": "plan",
-                "default_value": "lakehouse-enterprise",
-                "options": [
-                  {
-                    "displayname": "Lite",
-                    "value": "lite"
-                  },
-                  {
-                    "displayname": "Enterprise",
-                    "value": "lakehouse-enterprise"
-                  }
-                ]
-              },
-              {
-                "key": "resource_tags"
-              },
-              {
-                "key": "access_tags"
-              },
-              {
-                "key": "existing_kms_instance_crn"
-              },
-              {
-                "key": "existing_kms_key_crn"
-              }
-            ]
+            {
+              "key": "ibmcloud_api_key",
+              "required": true,
+              "type": "password"
```

Comment:
no need to add `required` or `type` here. Leave it out so it defaults to the terraform code settings. Otherwise we need to maintain in 2 places

Reply:
done